src/quilla/hookspecs/configuration.py

```diff
@@ -131,3 +83,17 @@ def quilla_resolve_enum_from_name(name: str, enum: Type[T]) -> Optional[T]:
         The resolved enum, if it can be resolved. None if the plugin can't resolve
         the value.
     '''
+
+
+@hookspec
+def quilla_prevalidate(validation: UIValidation):
```

Comment:
quilla_prevalidate can also be put in its own file to handle more generic cases.

Reply:
I think I'll migrate this one to a new file after more hooks are added, since there'll be a more distinct file that would make sense for it to be in that I haven't come up with the proper naming scheme for yet

Reply:
Added #22 to address this later